inbm/telemetry-agent/telemetry/poller.py

```diff
@@ -45,8 +45,8 @@ def __init__(self) -> None:
         self._lower_bound_max_cache_size = 50
         self._upper_bound_max_cache_size = 200
         self._upper_bound_swbom_interval_hours = 168
-        self._lower_bound_swbom_interval_hours = 24
-        self._swbom_interval_seconds = self._swbom_timer_seconds = 86400
+        self._lower_bound_swbom_interval_hours = 1
```

Comment:
@nmgaston it can be a value that the user might want to set(this is open-ended). Our default value doesn't change, just increased the lower limit value for making it easy for testing(the only reason why it's changed now)

Reply:
@tsirlapu - I'd like to understand more the scenario of why this is needed to speed up testing.  If it's just  for testing something now then I would just make the change temporarily and then change it back before merging it.  I don't see that someone would want to get the swbom information every hour.  It's not something that changes that frequently.

Reply:
@nmgaston agree with you(swbom won't change that frequently) let's discuss this tomorrow in the daily meeting